src/Microsoft.AspNet.Mvc.ModelBinding/Binders/CompositeModelBinder.cs

```diff
@@ -188,5 +190,11 @@ private static BodyBindingState GetBodyBindingState(ModelBindingContext oldBindi
 
             return state;
         }
+
+        private static BindingSource GetBindingSource(IBinderMetadata metadata)
```

Comment:
This will be part done in the next PR when it moves to model metadata. For now I'm consolidating all of these on BindingSource.